packages/contracts-bedrock/src/L2/GasPriceOracle.sol

```diff
@@ -179,6 +194,17 @@ contract GasPriceOracle is ISemver {
         return l1GasUsed + IL1Block(Predeploys.L1_BLOCK_ATTRIBUTES).l1FeeOverhead();
     }
 
+    function getOperatorFee(uint256 _gasUsed) public view returns (uint256) {
+        if (!isIsthmus) {
+            return 0;
+        }
+
+        return Arithmetic.saturatingAdd(
+            Arithmetic.saturatingMul(_gasUsed, IL1Block(Predeploys.L1_BLOCK_ATTRIBUTES).operatorFeeScalar()) / 1e6,
```

Comment:
Did we ultimately revert the saturating addition in the geth state transition function?

Reply:
https://github.com/ethereum-optimism/op-geth/pull/388/files

Reply:
Note that we showed in the op-geth PR that an overflow is impossible, assuming that `gasUsed` is < `uint64.max`.

Reply:
Ok, i think this is good how it is